scripts/knowledge_repo

```diff
@@ -80,13 +80,10 @@ args, remaining_args = parser.parse_known_args()
 if args.version:
     print('Local version: {}'.format(knowledge_repo.__version__))
 
-if args.help and not remaining_args:
-    parser.print_help()
-    sys.exit(0)
-
 if args.repo is None:
+    parser.print_help()
     raise ValueError("No repository specified. Please set the --repo flag, \
-                      or the KNOWLEDGE_REPO environment variable.")
+or the KNOWLEDGE_REPO environment variable.")
```

Comment:
We've settled on aligning everything with the opening delimiter (https://www.python.org/dev/peps/pep-0008/#indentation) - there are more examples of this in the actual app code.